### DISCUSSION TITLE
Which TCP port to use with UseNetTcp when hosting on IIS

### DISCUSSION THREAD

**Discussion by oskarjf** [CoreWCF/CoreWCF#1305] [Q&A]
I'm a bit unsure as to what the TCP port should be when hosting on IIS. I'm calling IWebHost.UseNetTcp(808) in the Startup.
If I set it to 808 which is the same port number I've defined for net.tcp in IIS I get an error when starting the application. 
Fails with: "Failed to bind to address http://0.0.0.0:808: address already in use."

.Net 7 / CoreWCF 1.4.1 / IIS v10

**Comment by gyurisc** [comment]
Did you try 8080 for HTTP? This is also something that you can configure for your HTTP binding.
